### PR TITLE
LPD-32793

### PR DESCRIPTION
**Pull request by ChrisKian**
Related issue: https://liferay.atlassian.net/browse/LPD-32793

Rebased and forwarded from https://github.com/liferay-appsec/liferay-portal/pull/1883

This PR doesn't have tests included because the changes from [LPD-6829](https://liferay.atlassian.net/browse/LPD-6829)/[LPD-23255](https://liferay.atlassian.net/browse/LPD-23255) will include full test coverage.  See [this smoke test](https://github.com/brianchandotcom/liferay-portal/pull/153369/commits/0b034661006758d1b757f8d64344cc19fe855f87#diff-6257e3a4c7b630ced44f04b270add1465e2b30d69d0adebc15848f7dc13fb41bR13-R25).

We can note this situation once forwarded to BChan.  Thanks @rafaprax!

CC @daiazius

### CODE REVIEW

**Comment by ChrisKian** [comment]
ci:test:relevant

File: modules/dxp/apps/scim/scim-configuration-web/src/main/resources/META-INF/resources/portal_settings/scim_configuration.jsp [hunk 1]
```diff
@@ -29,43 +29,47 @@ String oAuth2ApplicationName = GetterUtil.getString(request.getAttribute(ScimWeb
...
-		<div class="form-group">
-			<label for="<%= oAuth2AccessTokenInputId %>">
-				<liferay-ui:message key="access-token" />
-			</label>
-
-			<div class="input-group input-group-sm">
-				<div class="input-group-item input-group-prepend">
-					<input class="form-control" id="<%= oAuth2AccessTokenInputId %>" readonly value="<%= oAuth2AccessToken %>" />
+		<c:choose>
+			<c:when test="<%= Validator.isNotNull(oAuth2ApplicationName) %>">
```
Comment (rafaprax) [inline modules/dxp/apps/scim/scim-configuration-web/src/main/resources/META-INF/resources/portal_settings/scim_configuration.jsp:33]: Hey guys, as you have just one `c:when` statement, use a `c:if` instead. Thanks

**Review by rafaprax — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/scim/scim-configuration-web/src/main/resources/META-INF/resources/portal_settings/scim_configuration.jsp (hunk 1 above)
Comment (rafaprax) [reply modules/dxp/apps/scim/scim-configuration-web/src/main/resources/META-INF/resources/portal_settings/scim_configuration.jsp:33]: \cc @daiazius

**Comment by ChrisKian** [comment]
ci:test:sf

**Comment by ChrisKian** [comment]
ci:test:relevant

**Comment by ChrisKian** [comment]
ci:test:sf

**Comment by ChrisKian** [comment]
ci:test:relevant

**Comment by ChrisKian** [comment]
Hey @daiazius,
With the inclusion of LPD-23250, the `scim_configuration.jsp` file changes a bit.  I've added some changes as well, so could you please test them and ensure I didn't break anything?  Thanks!

**Comment by daiazius** [comment]
Hey @ChrisKian, Yes, the new logic is to hide the field and only show it after a configuration is saved. The test is broken now because after resetting you would need to save a configuration for the field to appear.

**Comment by ChrisKian** [comment]
ci:test:sf

**Comment by ChrisKian** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
Hey @ChrisKian, the code looks good to me. So, if there are no more test suites to run, feel free to forward this pr. Thank you

**Comment by ChrisKian** [comment]
ci:forward

**Pull request closed without merging**